pkg/client/unversioned/helper.go

```diff
@@ -552,6 +666,17 @@ func defaultVersionFor(config *Config) string {
 	return version
 }
 
+// defaultVersionForGroup is shared between defaultServerUrlFor and RESTClientFor
+func defaultGroupVersionForGroup(group string, config *Config) string {
```

Comment:
Add to the Godoc that this is specific to API groups defined in the Kube codebase, not those compiled outside it.